INSTALL.md

```diff
@@ -1,108 +1,92 @@
 # PyBitmessage Installation Instructions
 
-For an up-to-date version of these instructions, please visit the
-[Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
+## Download
```

Comment:
split installation into
- binary
  - windows (32bit only): point to download.bitmessage.org
  - linux (64bit): appimage.bitmessage.org
- source: git clone